packages/openapi-fetch/test/common/create-client.test.ts

```diff
@@ -57,6 +57,47 @@ describe("createClient options", () => {
     expect(actualURL.href).toBe("https://api.foo.bar/v2/resources");
   });
 
+  test("Request option is used", async () => {
```

Comment:
There is already a test for the client `test/common/request.test.ts`. Could you move (and probably align) the test for the case this PR is fixing there, please?